studentslist/lib/main.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+final List<int> students = <int>[for(var i=1; i<=100; i+=1) i];
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final title = 'Students List';
+
+    return MaterialApp(
+      title: title,
+      home: Scaffold(
+        appBar: AppBar(title: Text(title)),
+        
+        body: ListView.builder(
+  padding: const EdgeInsets.all(8),
+  itemCount: students.length,
+  itemBuilder: (BuildContext context, int index) {
+    return Container(
+      height: 50,
```

Comment:
Instead of height, please use margin. Margin is better because you never know if the text might wrap to 2 lines or more.

https://api.flutter.dev/flutter/widgets/Container-class.html#example

Reply:
done!